File: backend-springboot/src/main/java/com/example/backendspringboot/model/Trip.java
package com.example.backendspringboot.model;

import jakarta.persistence.Column;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name="trips")

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Trip {
    @Id
    private Long id;
    @Column(name = "from_city")
    private String fromCity;
    @Column(name = "to_city")
    private String toCity;
    private String date;
    private String time;
    private Double price;
    private String companyName;
}

File: backend-fastapi/model/Trip.py
from sqlalchemy import Column, Integer, String, Date
from database import Base

class Trip(Base):
    __tablename__ = "trips"

    id = Column(Integer, primary_key=True, index=True)
    from_city = Column(String)
    to_city = Column(String)
    date = Column(Date)
    price = Column(Integer)